bundle/src/test/java/com/adobe/acs/commons/errorpagehandler/cache/impl/PowerMockErrorPageCacheImplTest.java

```diff
@@ -39,8 +40,13 @@
  *
  * Split out as ErrorPageCacheImplTest was having problems with @Spy'ied vars
  */
+
+// See https://stackoverflow.com/questions/52966897/powermock-java-11 for details about the 
+// @PowerMockIgnore statement
+
 @RunWith(PowerMockRunner.class)
 @PrepareForTest(ResourceDataUtil.class)
+@PowerMockIgnore({"com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*", "org.w3c.*"})
```

Comment:
Yes this change works for sure, I've had to do this in other projects as well.  Ultimately however we might try to find alternatives to mocking static methods altogether where possible.